res/values-pt-rBR/strings.xml

```diff
@@ -1139,7 +1139,7 @@ Esta é uma lista de coisas que é * PROIBIDO * para usar nas skins personalizad
     <string name="x3_Clan_XP_Perm">3x Clã XP Permanente!</string>
     <string name="x2_Clan_XP_Perm">2x Clã XP Permanente!</string>
     <string name="Japan">Japão</string>
-    <string name="South_Korea">Coreia do Sul</string>
+    <string name="South_Korea">Coréia do Sul</string>
```

Comment:
According to Portuguese grammatical rules, "Coréia" isn't supposed to have an acute.